cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_utils.go

```diff
@@ -20,12 +20,19 @@ import (
 	"strconv"
 
 	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	nodeGroupMinSizeAnnotationKey = "machine.openshift.io/cluster-api-autoscaler-node-group-min-size"
 	nodeGroupMaxSizeAnnotationKey = "machine.openshift.io/cluster-api-autoscaler-node-group-max-size"
+
```

Comment:
can we update this annotations to match what's given here
openshift/cluster-api-provider-aws#301
openshift/cluster-api-provider-azure#112

Reply:
similar question to the other annotations, should we make these exportable from the autoscaler so that the providers can import them?

Reply:
i am seeing the comments from the other pr. no further clarification needed =)

Reply:
These `nodeGroupInstance...` annotations are being used to parse, but the `...Key` ones are what will exist, so I think the parsing needs updating and these 4 consts can be dropped (if I've understood correctly)

Reply:
that makes sense. thanks!